yarn.lock

```diff
@@ -2751,18 +2687,6 @@
     qs "^6.6.0"
     util-deprecate "^1.0.2"
 
-"@storybook/router@6.1.14":
```

Comment:
@aitchiss I have added the changed in yarn.lock but they actually look like storybook changes. It may not be related to this PR.